test/test_jit.py

```diff
@@ -4761,6 +4764,17 @@ def func(x, value1, value2):
 
             self._check_code(template.format(indexing), "func", [tensor, value1, value2])
 
+        # Torchscript assumes type Tensor by default, so we need this explicit
+        # declaration.
+        def check_indexing_list_int(indexing, list):
```

Comment:
Can we add some negative str slicing tests ?

Reply:
yep